python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -537,7 +550,8 @@ class RandomForestRegressor(Base):
         return predicted_result
 
     def predict(self, X, predict_model="GPU",
-                algo='BATCH_TREE_REORG', convert_dtype=True):
+                algo='BATCH_TREE_REORG', convert_dtype=True,
+                fil_sparse_format=False):
```

Comment:
If we're introducing this parameter, the default value `'auto'` makes more sense, as the default values will defer the decision on the algorithm and forest format to the implementation.